voc/no_authentication.go

```diff
@@ -28,3 +28,7 @@ package voc
 type NoAuthentication struct {
 	*Authenticity
```

Comment:
Just thinking out loud: Couldn't we leave this out since we know there is no authentication when no other authentication is in place? I have to look at the metrics how its done there. For that we would need some kind of "authentication" functionality that can contain multiple authentication methods or sth. similar.

Reply:
Needs to be resolved in the ontology.